### DISCUSSION TITLE
Tangle to path with sudo

### DISCUSSION THREAD

**Discussion by refaelsh** [nvim-neorg/neorg#1241] [Q&A] [answered]
I was wandering, how to tangle to a path that requires sudo? For example, to `/etc/nixos/configuration.nix`.

**Comment by andreadev-it** [comment, marked as answer]
I guess that you would have to open neovim as an administrator to do that. Have you tried it?

**Reply by refaelsh** [reply]
Damn... Should have thought about this myself. I feel a bit ashamed :-) 
> Have you tried it?

Tried it just now and it works.

Thanks!

**Reply by andreadev-it** [reply]
Nothing to feel ashamed for :)
Glad it worked, have a nice day
